### PR TITLE
SC-12339: NewRelic - ARM architecture support

### PR DESCRIPTION
**Pull request by zyuzka**
## PR Description

Tiket: https://spryker.atlassian.net/browse/SC-12339

## Change log

- added Newrelic for ARM64
- updated PHP versions
- removed PHP 7.4

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by kalaputsik — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -33,17 +33,17 @@ Extends official PHP Docker images with extensions and tools to be able to run S
...
+| [spryker/php:8.0-alpine3.15](https://hub.docker.com/r/spryker/php/tags?name=8.0-alpine3.15)       | 8.0.26      | Alpine 3.15        | [![](https://images.microbadger.com/badges/image/spryker/php:8.0-alpine3.15.svg)](https://microbadger.com/images/spryker/php:8.0-alpine3.15 "Get your own image badge on microbadger.com") | [:link:](https://github.com/spryker/docker-php/blob/master/alpine/3.15/8.0/Dockerfile)     |
 | [spryker/php:7.4-alpine3.15](https://hub.docker.com/r/spryker/php/tags?name=7.4-alpine3.15)       | 7.4.33      | Alpine 3.15        | [![](https://images.microbadger.com/badges/image/spryker/php:7.4-alpine3.15.svg)](https://microbadger.com/images/spryker/php:7.4-alpine3.15 "Get your own image badge on microbadger.com") | [:link:](https://github.com/spryker/docker-php/blob/master/alpine/3.15/7.4/Dockerfile)     |
```
Comment (alexanderM91) [inline README.md:41]: Could you please remove PHP 7.4 from the ReadMe?

File: context/php/disabled/newrelic.ini [hunk 2]
```diff
@@ -4,6 +4,7 @@ newrelic.license = ${NEWRELIC_LICENSE}
 newrelic.appname = ${NEWRELIC_APPNAME}
 newrelic.framework = "no_framework"
 newrelic.browser_monitoring.auto_instrument = false
+newrelic.daemon.location = "/usr/bin/newrelic-daemon"
```
Comment (alexanderM91) [inline context/php/disabled/newrelic.ini:7]: Why do we need to introduce this options?

File: debian/bullseye/8.0/Dockerfile [hunk 3]
```diff
@@ -126,13 +134,23 @@ ENV NEWRELIC_DISTRIBUTED_TRACING_EXCLUDE_NEWRELIC_HEADER=0
...
+    && export NR_INSTALL_SILENT=1 \
+    && /tmp/newrelic-php5-*/newrelic-install install \
+    && rm -rf /tmp/newrelic-php5-* /tmp/nrinstall* \
+    && /usr/bin/install -d -m 777 /var/log/newrelic/ \
+    && rm /usr/local/etc/php/conf.d/newrelic.ini \
+    && ln -s /lib64/ld-linux-x86-64.so.2 /lib/ld-linux-x86-64.so.2; fi'
+
+COPY --from=spyker-newrelic-arm /newrelic-daemon /tmp/newrelic-daemon
+
+RUN bash -c 'if [ "${TARGETPLATFORM}" == "linux/arm64" ]; then \
```
Comment (alexanderM91) [inline debian/bullseye/8.0/Dockerfile:147]: Can we handle this staff on the `spyker-newrelic-arm` side?

File: context/php/disabled/newrelic.ini (hunk 2 above)
Comment (zyuzka) [reply context/php/disabled/newrelic.ini:7]: Latest Agent doesn't work without daemon

File: debian/bullseye/8.0/Dockerfile (hunk 3 above)
Comment (zyuzka) [reply debian/bullseye/8.0/Dockerfile:147]: Nope. I tried to use an agent from `spyker-newrelic-arm` based on Aws Linux and Debian. And every time, we had an error with `libpcre`. For Debian, I found this solution: compile and install the agent directly with `make agent-install`

**Pull request merged**